client/my-sites/google-my-business/select-location/index.js

```diff
@@ -62,11 +69,17 @@ class GoogleMyBusinessSelectLocation extends Component {
 					{ translate( 'Select the listing you would like to connect to:' ) }
 				</CompactCard>
 
-				{ locations.map( ( location ) => (
-					<GoogleMyBusinessLocation
-						key={ location.id }
-						location={ location }
-					/>
+				{ locations.map( location => (
+					<CompactCard className="gmb-location">
+						<GoogleMyBusinessLocation key={ location.id } location={ location } />
+						<Button
+							className="gmb-location__button"
+							href={ `/stats/${ siteSlug }` }
+							onClick={ this.trackConnectLocationButtonClick }
+						>
+							{ translate( 'Connect Location' ) }
+						</Button>
+					</CompactCard>
```

Comment:
Maybe passing a button to the `GoogleMyBusinessLocation` component as a child (i.e. using the `children` prop) could have allowed us to keep all styles and markup in this component while allowing this kind of flexibility.